docs/releasing/publishing-a-pre-release.md

```diff
@@ -46,20 +43,83 @@ Projects can install `govuk-frontend@beta` in their package.json alongside other
+   - saving your changes
+
+8. Apply the new pre-release version number by running:
 
    ```shell
-   npm version <PRE-RELEASE TYPE> --preid <PRE-RELEASE TAG> --no-git-tag-version --workspace govuk-frontend
+   npm version <PRE-RELEASE TYPE> --preid <PRE-RELEASE IDENTIFIER> --no-git-tag-version --workspace govuk-frontend
    ```
 
-   This step will update the [`package.json`](/package.json) and project [`package-lock.json`](/package-lock.json) files.
+   This step will update [`govuk-frontend`'s `package.json`](/packages/govuk-frontend/package.json) and project [`package-lock.json`](/package-lock.json) files.
 
    Do not commit the changes.
 
-8. Run `npm run pre-release` to create and push a new branch that contains your changes. This process may take a few moments and will display a `Success!` message.
+9. Update browser data from ["Can I use"](https://caniuse.com) by running:
+
+   ```shell
+   npx update-browserslist-db@latest
+   ```
+
+   This step will update the project [`package-lock.json`](/package-lock.json) file if updates are found.
+
+10. Run `npm run build-release` to:
+
+    - build GOV.UK Frontend into [the package's `/dist`](/packages/govuk-frontend/dist) and [root `/dist`](/dist) directories
+    - commit the changes
+    - push a branch to GitHub
+
+    You will now be prompted to continue or cancel.
+
+11. If you're publishing a beta pre-release, create a pull request
+    When reviewing the PR, check that the version numbers have been updated and that the compiled assets use this version number.
+
+12. If you're publishing a beta pre-release, once a reviewer approves the pull request, merge it to **main**.
+
+## Publish a release to npm
+
+1. If you're publishing a beta pre-release, check out the **main** branch and pull the latest changes.
+
+2. Sign in to npm (`npm login`), using the credentials for the govuk-patterns-and-tools npm user from Bitwarden.
+
+3. Run `npm run publish-release`, which will prompt you to check whether the npm tag looks as expected.
+
+4. Enter `N` to continue to set the npm tag corresponding to the kind of release you're publishing:
+
+   - `internal` for internal pre-releases
+   - `next` for beta pre-releases
+
+5. You will now be prompted to continue or cancel the release. Check the details and enter `y` to continue. If something does not look right, press `N` to cancel the release.
+
+   This step will create a ZIP file containing the release in the root of your govuk-frontend git directory. You will need this file when creating the GitHub release.
+
+   It will also automatically create a tag in Github which you can use to create a Github release in the following section.
+
+6. Verify the presence of the pre-release and its tag on [npm](https://www.npmjs.com/package/govuk-frontend?activeTab=versions)
+
+   If the pre-release has been assigned the wrong tag (mistakes happen),
+   you can use [`npm dist-tag`](https://docs.npmjs.com/cli/v8/commands/npm-dist-tag) to quickly correct.
```

Comment:
**note** Figured a little final check wouldn't hurt. Also, as suggested by Ollie, dug how we can correct any mistakes that may have happened in tagging the package, while still logged in to npm. Also updated `publishing.md`.